server/s3_get_object_action.cc

```diff
@@ -183,6 +258,154 @@ void S3GetObjectAction::set_total_blocks_to_read_from_object() {
   }
 }
 
+void S3GetObjectAction::set_total_blocks_to_read_from_fragmented_object() {
+  // to read complete object, total number blocks to read is equal to total
```

Comment:
I think the comment does not make sense !!! Something missed in the sentence ?

Reply:
ok. Will remove that comment.

Reply:
Updated the comment and set it at the correct code line